server/build/cancel.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2022 Woodpecker Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
```

Comment:
Should we take this chance to rename `build`? To something like `pipeline`.

Reply:
And `queueBuild` to `queuePipeline`

Reply:
well if we call it pipeline it should go into package pipeline ;)

I might look into moving code to pipeline ...